components/apimgt/org.wso2.carbon.apimgt.rest.api.commons/src/main/java/org/wso2/carbon/apimgt/rest/api/common/util/RestApiUtil.java

```diff
@@ -439,4 +479,33 @@ public static String getHost(String protocol) {
                     ExceptionCodes.POLICY_LEVEL_NOT_SUPPORTED);
         }
     }
+
+    /**
+     * Converts time in millis to ISO-8601 date time format
+     *
+     * @param epochMilliSeconds time in milli seconds
+     * @return time in ISO 8601 format
+     */
+    public static String epochToISO8601DateTime(long epochMilliSeconds) {
```

Comment:
I think these two methods should move to core

Reply:
This method is used only in REST APIs.